WordPress/Classes/ViewRelated/Basic Profile/BasicUserProfileViewController.swift

```diff
@@ -33,41 +39,67 @@ final class BasicUserProfileViewController: UIViewController, DrawerPresentable,
 
             self.nameLabel.text = profile?.formattedName
             self.usernameLabel.text = "@\(profile?.preferredUsername ?? "")"
-            if let location = profile?.currentLocation {
+            if let location = profile?.currentLocation,
+                !location.isEmpty {
                 self.locationLabel.text = location
             } else {
                 self.locationLabel.isHidden = true
             }
-            if let aboutMe = profile?.aboutMe {
+            if let aboutMe = profile?.aboutMe,
+                !aboutMe.isEmpty {
                 self.aboutMeLabel.text = aboutMe
             } else {
                 self.aboutMeLabel.isHidden = true
             }
 
-            self.contentView.stopGhostAnimation()
-
-            // TODO - BASICPROFILES: Update sites
+            self.sites = profile?.urls ?? []
+            self.stopGhostAnimations()
         }
     }
 
     override func viewWillAppear(_ animated: Bool) {
         super.viewWillAppear(animated)
         if isInitialDisplay {
             contentView.startGhostAnimation()
+            updateGhostableTableViewOptions(cellClass: SiteTableViewCell.self, identifier: "Sites")
             isInitialDisplay = false
         }
     }
 
+    private func configureSitesTableView() {
+        sitesTableView.dataSource = self
+        sitesTableView.register(WPBlogTableViewCell.self,
+                                forCellReuseIdentifier: WPBlogTableViewCell.reuseIdentifier())
+        sitesTableView.accessibilityIdentifier = "Sites"
```

Comment:
Copied this value from the page it's being used currently. Seems appropriate to me.